### PR TITLE
Introduce GitHub issue forms

### LINKED ISSUE #585: Provide labels at issue templates?
**How would this feature be useful?**

Currently, when a user creates an issue, it doesn't has labels. Maybe we can alleviate the maintainer's work by automating the labels addition.

**Describe the solution you'd like**

Add labels to the issue templates: `bug` at the bug template and `enhancement` at the feature request, maybe?

**Describe alternatives you've considered**

If we keep as is, it's ok for me too :+1:

### PR DESCRIPTION
**Pull request by FollowTheProcess**
Closes #585 

I've taken an initial stab at porting our issue templates to the new [GitHub forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates).

I've tried to keep the intent of the originals but also threw some of the nice features of the new templates at them for good measure!

I've also added a new form for a generic question about Nox as I believe this is useful for people who want to know something but don't want to raise a "scary" sounding bug or issue.

More than happy to tweak the contents if we want, just kicking things off 👍🏻 

You can view the new forms by clicking on the files and GitHub will render it how it will look when filing the issue 🙂

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: ❓ Question
```
Comment (DiddiLeija) [inline .github/ISSUE_TEMPLATE/question.yml:1]: Are questions encouraged in this repo? Maybe I misunderstood #534, but maybe the Winterbloom Discord server (under the `#nox` channel) could fit better for questions.

However, we could keep this template, and also provide a link in a `ISSUE_TEMPLATE/config.yml` [^1] file for those who want to ask on DIscord.

Just tagging @wntrblm/nox to attract the maintainers' attention, to get some more opinions.

[^1]: `config.yml` reference is [here at the GitHub docs](https://docs.github.com/es/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

**Review by DiddiLeija — COMMENTED** [review]
Strong +1 from me! I watched how these new templates worked in [pip](https://github.com/pypa/pip) and some other projects, and I think they will look pretty nice here!

File: .github/ISSUE_TEMPLATE/question.yml (hunk 1 above)
Comment (FollowTheProcess) [reply .github/ISSUE_TEMPLATE/question.yml:1]: I like that idea!

Comment (DiddiLeija) [reply .github/ISSUE_TEMPLATE/question.yml:1]: I think I can write a `config.yml` after this PR, unless someone disagrees :)

**Review by crwilcox — APPROVED** [review]
+1 on this. Only comment is if we wouldn't benefit from making some fields required. But this is the closest to a port, by not doing that, and perhaps worth further discussion. 

Thanks for taking this on.

**Comment by FollowTheProcess** [comment]
I've removed the question form from this as we're planning to include a permalink to the nox discord channel as discussed in #534

**Comment by FollowTheProcess** [comment]
@crwilcox I think a sensible thing to do with making things required might be to use these templates for a period of time and see if new issues are lacking important information, then we can make those fields required or adjust other aspects of the forms going forward?

**Pull request merged**